docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: '3.8'
```

Comment:
Looks a little dangerous; could you elaborate on this fix a bit?

Reply:
I get this warning every time I run something using docker:

```
WARN[0000] /Users/ha/Code/php-data-objects/docker-compose.yml: `version` is obsolete
```

I removed it from StatMando a couple of weeks ago and it's been fine.  Googled this: https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313

Reply:
Same is happening in PHP-API but I haven't changed it there.

Reply:
Sounds good. We'll need to be careful with PHP-API since we deploy the PHP-API docker image; we don't deploy this repo so it should be fine.

Reply:
I didn't know that the php-api docker image was used for both dev and prod. That's good to know! It shouldn't need to change much, though.